content/200-concepts/300-more/050-environment-variables/index.mdx

```diff
@@ -7,7 +7,7 @@ tocDepth: 2
 
 <TopBlock>
 
-You can use environment variables to provide configuration options when a CLI command is invoked. Environment variables allow you to:
+You can use environment variables to provide configuration options when a CLI or Client command is invoked. When CLI commands are used, environment variables allow you to:
```

Comment:
This is also close to where we should probably link to the `env()` concept page that explains how to use them in the schema to do things.